docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,395 @@
+# Drupal Configuration Provided By Islandora Defaults
+
+This is a list of the configuration items provided by the Islandora Defaults module. 
+
+## Entities
+
+(see Glossary: [Entity](../user-documentation/glossary.md#entity))
+
+### Content Types (i.e. Node bundles)
+
+|Name  					|Description  		 		|Comments							|
+|---					|---					|---								|
+|Repository Item (islandora_object)   	|An item in your Islandora repository.  |Defaults provides only one node type to use with Islandora, and different behaviours are triggered by different field values. If you would like to use another content type with Islandora, it will need Islandora behaviours configured appropriately (see below). 	|
+
+
+### Media Types (i.e. Media bundles)
+
+|Name  					|Description  		 		|Comments							|
+|---					|---					|---								|
+|Audio (audio)   	| A locally hosted audio file.  	| This is Drupal's built-in Audio media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
+|Document (document) 	| An uploaded file or document, such as a PDF.		| This is Drupal's built-in Document media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.|
+|File (file)		| Use local files for reusable media.			||
+|Image (image)		| Use local images for reusable media.			| This is Drupal's built-in Image media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
+|Video (video)		| A locally hosted video file.				| This is Drupal's built-in Video media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
+
+### Vocabularies (i.e. Taxonomy bundles)
+
+|Name			|Description		|Comments			|
+|---			|---			|---				|
+|Islandora Access (islandora_access)	|Terms used to limit, restrict or coordinate access	| Not sure if they actually do affect access.	|
```

Comment:
Instead of `Not sure if they actually do affect access.`, how about `Meant to be used with permissions_by_term. The vocabulary is unpopulated by default.`

Reply:
Sounds good.